src/main/scala/fundamentals/level03/TryExercises.scala

```diff
@@ -97,23 +96,20 @@ object TryExercises {
   trait Employee
 
   /**
-    * 1. Now remove `import TryTestTypes._` from `TryExercisesTest.scala`
-    * 2. Comment out the contents of `TryTestTypes`
+    * Now remove `import TryTestTypes._` from `TryExercisesTest.scala`
```

Comment:
I think we still need to comment out the content of `TryTestTypes`. Or is this not the case anymore? @wjlow

Reply:
AFAICT, it compiles when the import is removed, but the contents are left uncommented (at least on my machine). But would be good if someone else can confirm this.

Reply:
@chanjk I think this example doesn’t need you to comment out the content of the file. 

It’s just LogParser because of the sealed trait that can no longer be extended from.